### ISSUE TITLE
[Bug] Splinter Cell: Conviction fails to launch

### ISSUE THREAD

**Issue by NikolaiX** [ZeroFox5866/nucleuscoop#52]
I've just downnloaded the latest version of the app (1.1.0) and attempted to hook Splinter Cell Conviction. When I hit play, a cmd window opens with the application path, then closes. The game doesn't launch, and after about 20 seconds, Nucleus itself silently closes. Downloading a fresh copy of the game from Uplay didn't fix the issue.

Windows Event Log
---------------------------------------------------------------------
Application: NucleusCoop.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ComponentModel.Win32Exception
   at System.Diagnostics.ProcessManager.OpenProcess(Int32, Int32, Boolean)
   at System.Diagnostics.Process.GetProcessHandle(Int32, Boolean)
   at System.Diagnostics.Process.get_HasExited()
   at Nucleus.Gaming.GenericGameHandler.Update(Double)
   at Nucleus.Coop.MainForm.UpdateGameManager(System.Object)
   at System.Threading.ThreadHelper.ThreadStart_Context(System.Object)
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Threading.ThreadHelper.ThreadStart(System.Object)
---------------------------------------------------------------------

I've done some digging and believe the issue relates to the JDiff process used by the script: manually running the included .bat file in cmd outputs the following, so looks like an issue with the .jdf file:
---------------------------------------------------------------------
C:\NucleusCoop\content\Splinter Cell Conviction\Instance0\src\system>jdiff.exe -u conviction_gameog.exe conviction_game.jdf conviction_game.exe
Error reading source file.

Error reading file !
---------------------------------------------------------------------

**Comment by hmoda778** [comment]
the same problem with me 😔

**Comment by Talos910** [comment]
Yeah the .jdf patch only works with the steam version exe, I assumed the exe was the same for both the steam and uplay version cause both need uplay but that does not seem to be the case.

**Comment by hmoda778** [comment]
اI downloaded a steam version of the game.. And the same problem still .. What is the solution in your opinion

**Comment by NikolaiX** [comment]
> Yeah the .jdf patch only works with the steam version exe, I assumed the exe was the same for both the steam and uplay version cause both use uplay but that does not seem to be the case.

> اI downloaded a steam version of the game.. And the same problem still .. What is the solution in your opinion

I'm wondering if there's regional differences in the executables or something that would cause the .jdf to fail? @Talos910 would it be useful if I sent you the Uplay version of the executable for comparison?

**Comment by Talos910** [comment]
Just tested latest legit steam version and it is launching fine, make sure your antivirus or windows defender is not blocking any of the script files. Yeah could probably include a patch for the uplay version with the exe, you can dm it in the Nucleus discord if you want.

**Comment by NikolaiX** [comment]
> Just tested latest legit steam version and it is launching fine, make sure your antivirus or windows defender is not blocking any of the script files. Yeah could probably include a patch for the uplay version with the exe, you can dm it in the Nucleus discord if you want.

Discord dm sent.
